.github/workflows/package.yml

```diff
@@ -10,6 +10,45 @@ on:
     - cron: |
         0 0 * * *
 jobs:
+  source:
+    name: Source
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt -y -V install \
+            qemu-user-static \
+            ruby
+          sudo gem install bundler
+          bundle install
+      - name: Clone dependencies
+        run: |
+          cd ..
+          git clone --depth 1 https://github.com/groonga/groonga.git
+          cd groonga
+          git submodule update --init --force --recursive --depth 1
```

Comment:
Could you use the latest released archive instead?

Reply:
In my second thought, the archive file doesn't depend on groonga.
So I have remove this step.

Related to this, the linux job use not the latest released archive but the latest Groonga repository.

https://github.com/pgroonga/pgroonga/blob/master/.github/workflows/package.yml#L88

I will fix it in another PR.